packages/token-utils/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@p2p-wallet-web/token-utils",
+  "version": "1.0.0",
+  "license": "MIT",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "build": "rm -fr dist/ && tsc && tsc -P tsconfig.cjs.json",
+    "watch": "rm -fr dist/ && tsc && tsc -P tsconfig.cjs.json -w"
+  },
+  "dependencies": {
+
```

Comment:
Don't see @saberhq/token-utils in dependencies and in peerDependecies, but it is used in code